.gitignore

```diff
@@ -7,6 +7,7 @@ yarn-error.log*
 lerna-debug.log*
 .pnpm-debug.log*
 
+test.md
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 
```

Comment:
- [ OPTIONAL] I noticed that you have been working on a Git repository and might need some help with managing your files. If you are not aware, Git allows you to ignore specific files from being tracked, and this can be useful when you want to keep certain files out of version control. Kindly I don't get the reason why you are ignoring these ? why this  `# Diagnostic reports (https://nodejs.org/api/report.html)
report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json` and `lerna-debug.log*
.pnpm-debug.log*`